main.py

```diff
@@ -1,53 +1,58 @@
-import cv2, numpy as np, time, sys
+import time
+import sys
```

Comment:
@guinnesschen It is better to import one module per line. Please refer to PEP8 (https://www.python.org/dev/peps/pep-0008/#imports)